plugin/trino-kudu/pom.xml

```diff
@@ -217,6 +229,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.rnorth.duct-tape</groupId>
+            <artifactId>duct-tape</artifactId>
+            <version>${dep.duct-tape.version}</version>
```

Comment:
We should add this to the root `dependencyManagement` section. (and remove the version from the other modules)